config/environments/production.rb

```diff
@@ -69,6 +69,23 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
 
+  Rails.application.routes.default_url_options[:host] = 'share-necessities.herokuapp.com'
+  config.action_mailer.default_url_options = { :host => 'share-necessities.herokuapp.com' }
```

Comment:
Line is too long. [91/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.